packages/core/core/test/AssetGraphBuilder.test.js

```diff
@@ -35,7 +35,8 @@ const DEFAULT_OPTIONS = {
   entries: [],
   logLevel: 'none',
   rootDir: FIXTURES_DIR,
-  targets: []
+  targets: [],
```

Comment:
I wish we configured prettier to do trailing commas. That would get rid of a lot of these diffs 😢

Reply:
cc @devongovett @DeMoorJasper to bikeshed this

I'd love to make this happen once PRs and Parcel 1 activity calms down